Introduced 2 new variables (#6159)
pingcap/docs: system-variables.md

- This variable is used to set whether the optimizer executes the optimization operation of pushing down the aggregate function to the position before Join, Projection, and UnionAll.
- When the aggregate operation is slow in query, you can set the variable value to ON.

> ### `tidb_opt_limit_push_down_threshold`

Reviewer: ```suggestion
### tidb_opt_limit_push_down_threshold
```

- Scope: SESSION | GLOBAL
- Default value: `100`
- Range: `[0, 2147483647]`

> - This variable is used to push down the Limit and TopN operators to the threshold in TiKV.

Reviewer 2: `This variation is used to set the threshold that determines whether push the Limit or TopN operators down to TiKV`

Reviewer 2: Rest LGTM

Reviewer: ```suggestion
- This variable is used to set the threshold that determines whether to push the Limit or TopN operator down to TiKV.
```

> - If the value of Limit or TopN is less than or equal to this threshold, then the Limit and TopN operators are forcibly pushed down to TiKV. This variable provides a solution when Limit or TopN cannot be pushed down due to estimation errors. 

Reviewer: ```suggestion
- If the value of the Limit or TopN operator is smaller than or equal to this threshold, these operators are forcibly pushed down to TiKV. This variable resolves the issue that the Limit or TopN operator cannot be pushed down to TiKV partly due to wrong estimation. 
```

> ### `tidb_opt_enable_correlation_adjustment`

Reviewer: ```suggestion
### tidb_opt_enable_correlation_adjustment
```

- Scope: SESSION | GLOBAL
- Default value:  `ON`

> - This variable is used for the optimizer whether to enable the method that estimates the number of rows based on column order correlation

Reviewer: ```suggestion
- This variable is used to control whether the optimizer estimates the number of rows based on column order correlation
```
